routes/post.go

```diff
@@ -958,6 +963,129 @@ func (fes *APIServer) GetPostsStateless(ww http.ResponseWriter, req *http.Reques
 	}
 }
 
+// GetPostsHashlistRequest ...
+type GetPostsHashlistRequest struct {
+	PostsHashlist              []string `safeForLogging:"true"`
```

Comment:
it's really a post hash hex list - we should be explicit about what we expect to be passed here. I think this also applies to the route and function names

Reply:
Agreed, changed the name everywhere